scripts/generate-available-permission-types-table.py

```diff
@@ -46,31 +47,37 @@
     ResourceType.TRACE,
 ]
 
+def add_lines_for_permission_set(resource_title, permission_types, lines):
+    lines.append('%s' % (resource_title))
+    lines.append('~' * len(resource_title))
+    lines.append('')
+
+    rows = []
+    rows.append(TABLE_HEADER)
+
+    for permission_type in permission_types:
+        description = PermissionType.get_permission_description(permission_type)
+        rows.append(['**%s**' % (permission_type), description])
```

Comment:
I prefer reading f strings. 😉 
```suggestion
        rows.append([f"**{permission_type}**", description])
```